docs/components/datetimepicker.md

```diff
@@ -0,0 +1,245 @@
+      calendarMode="time"
+      :isRange="false"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ JSON.stringify(currentValue1) }}</pre></div>
+  </template>
+</KCard>
+
+### Single date and time
+
+<KCard>
+  <template v-slot:body>
+    <KDateTimePicker
+      @changed="newVal => currentValue1 = newVal"
+      defaultMessage="Please select a time"
+      calendarMode="dateTime"
+      :isRange="false"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ JSON.stringify(currentValue1) }}</pre></div>
+  </template>
+</KCard>
+
+### Custom + Relative date/time ranges
+
+Display both a calendar and relative time frames, passing in a preset week-long range.
+
+<KCard>
+  <template v-slot:body>
+    <KDateTimePicker
+      @changed="newVal => currentValue1 = newVal"
+      defaultMessage="Please select a time range"
+      calendarMode="dateTime"
+      :isRange="true"
+      :timePeriods="exampleTimeFrames"
+      :defaultCustom="defaultRangeWeek"
+      v-model="currentValue1"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ JSON.stringify(currentValue1) }}</pre></div>
+  </template>
+</KCard>
+
+```html
+<KDateTimePicker 
+  @change="newVal => currentValue = newVal"
+  calendarMode="dateTime"
+  :defaultCustom="defaultRangeWeek"
+  :timePeriods=[...]
+/>
+```
+
+### Relative - show only time frame buttons
+
+Passing in a selected relative time frame value will default the component to default to the "Relative" tab.
+
+<KCard>
+  <template v-slot:body>
+    <KDateTimePicker
+      @changed="newVal => currentValue2 = newVal"
+      defaultMessage="Please select a time range"
+      :timePeriods="exampleTimeFrames"
+      :defaultRelative="defaultRelativeTimeframe"
+      v-model="currentValue2"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ JSON.stringify(currentValue2) }}</pre></div>
+  </template>
+</KCard>
+
+```html
+<KDateTimePicker
+  @changed="newVal => currentValue = newVal"
+  defaultMessage="Please select one of the time ranges below"
+  :timePeriods="exampleTimeFrames"
+  :defaultRelative="defaultRelativeTimeframe"
+/>
+```
+
+### Custom ranges - show only the calendar
+
+Calendar Display only - when the `timePeriods` prop is omitted, only the **Custom** tab is shown.
+
+<KCard>
+  <template v-slot:body>
+    <KDateTimePicker
+      @changed="newVal => currentValue3 = newVal"
+      defaultMessage="Please select some dates"
+      calendarMode="date"
+      :defaultCustom="defaultRangeTwoDay"
+      v-model="currentValue3"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ JSON.stringify(currentValue3) }}</pre></div>
+  </template>
+</KCard>
+
+```html
+<KDateTimePicker
+  @changed="newVal => currentValue = newVal"
+  defaultMessage="Please select some dates"
+  :defaultCustom="{
+    start: new Date().getTime(),
+    end: new Date().getTime() - (2*24*60*60*1000)
+  }"
+/>
+```
+
+## Props
+
+### calendarMode
+
+- Allowed values are: `date`, `time`, `dateTime`
+
+### isRange
```

Comment:
We want to move away from prefixing prop names with `is-` and `has-` so can we change this (and its usage in the component) to just `range`
```suggestion
### range
```